lib/restful_resource/http_client.rb

```diff
@@ -69,15 +69,18 @@ def message
       end
     end
 
+    DEFAULT_TIMEOUT = 10 # seconds
```

Comment:
If we add unit into the constant name `_S` we will be able to remove the comment
```suggestion
    DEFAULT_TIMEOUT_S = 10
```

Reply:
are we doing this in constants as well??
I agree on the env vars, since they are shared and have to be stack/tech agnostic, but in the code it should be fairly straight forward, since it's directly used in faraday
honestly I was even going to not put the comment :P

Reply:
I would like to see clearly what unit is that value in. Thanks to your comment `# seconds` I knew it was in seconds. When the constant is used it carries on that unit wherever.